dbt_sugar/core/yaml/yaml_helpers.py

```diff
@@ -0,0 +1,27 @@
+"""Contains yaml related utils which might get used in places."""
```

Comment:
also, the folder YAML can create complications with the module yaml.

Reply:
good catch, I've changed where the yaml should live. Coming to think of it I think it makes sense to have it under `clients` since it is how we interact with other systems I think.